design/apidsl/metadata.go

```diff
@@ -18,6 +18,11 @@ import (
 //
 //        Metadata("struct:field:name", "MyName")
 //
+// `struct:field:type`: overrides the Go struct field type generated by default by goagen.
+// Applicable to attributes only.
```

Comment:
Maybe add `The second optional tag value specifies the Go import path to the package defining the type if not built-in`.